content/en/docs/concepts/services-networking/service-traffic-distribution.md

```diff
@@ -0,0 +1,128 @@
+---
+reviewers:
+- gauravkg
+title: Service Traffic Distribution
+content_type: concept
+weight: 130
+description: >-
+  The `spec.trafficDistribution` field within a Kubernetes Service allows you to
+  express routing preferences for service endpoints. This can optimize network
+  traffic patterns for performance, cost, or reliability.
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.30" state="alpha" >}}
```

Comment:
Even better:
```suggestion
{{< feature-state feature_gate_name="ServiceTrafficDistribution" >}}
```

Reply:
(this automatically works out the right phase and which release it changed to the current phase)